### PR TITLE
Update server.js

### PR DESCRIPTION
**Pull request by johnsta**

### CODE REVIEW

File: Bikes/server.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ function requestIDParser(req, res, next) {
 // api ------------------------------------------------------------
 
 // find bike ------------------------------------------------------------
-app.get('/api/availableBikes', function (req, res) {
+app.get('/api/availableBikes1', function (req, res) {
```
Comment (DrEsteban) [inline Bikes/server.js]: >1 [](start = 28, length = 1)

I hate this change it offends me on a deeply personal level REJECTED

**Review by DrEsteban — CHANGES_REQUESTED** [review]
:clock1:

File: Bikes/server.js (hunk 1 above)
Comment (daniv-msft) [reply Bikes/server.js]: :smile: 

---
In reply to: [276480288](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/37#discussion_r276480288) [](ancestors = 276480288)

Comment (johnsta) [reply Bikes/server.js]: Get used to it! Apparently, my demos are now about breaking code. ;-)

**Review by daniv-msft — APPROVED** [review]
:shipit:

**Pull request closed without merging**